OEL-303: Update oe_showcase_page README.md (#23)
openeuropa/oe_showcase: modules/oe_showcase_page/README.md

# OpenEuropa Showcase Page

> The OpenEuropa Showcase Page is part of the Showcase profile implementation, it amins to demo several fetures of the [OpenEuropa Showcase](https://github.com/openeuropa/oe_showcase) module and [OpenEuropa Whitelabel](https://github.com/openeuropa/oe_whitelabel) by providing a content type to create pages with multiple content configuration using several paragraphs components that can be placed in any order.

Reviewer: ```suggestion
The OpenEuropa Showcase Page is part of the Showcase profile implementation, it aim is to demo several features of the [OpenEuropa Showcase](https://github.com/openeuropa/oe_showcase) module and [OpenEuropa Whitelabel](https://github.com/openeuropa/oe_whitelabel) by providing a content type to create pages using several paragraph components that can be placed in any order.
```

> The available paragraphs types are:

Reviewer: ```suggestion
The available paragraph types are:
```

- Quote
- Rich text
- Text with Featured media
- Document

## Dependencies

- [Paragraphs](https://www.drupal.org/project/paragraphs)
- [OpenEuropa Paragraphs](https://github.com/openeuropa/oe_paragraphs)
- [OpenEuropa Media](https://github.com/openeuropa/oe_media)

## Related modules

The related modules below provide demo content and configuration for the oe_showcase_page module:

- [OpenEuropa Showcase Navigation](https://github.com/openeuropa/oe_showcase): Together with OpenEuropa Showcase Default Content provides default configuration of menu links to each showcase page.
- [OpenEuropa Showcase Default Content](https://github.com/openeuropa/oe_showcase): Provides default demo content with several pages demonstrating the use of paragraphs. This module is based on the [Default Content](https://www.drupal.org/project/default_content) module, for further instructions on how to export/import content please refer to its page for updated information.

## Installation

Enable this module using Drush or normally via the Drupal Module management list.

In the root of the project, run

```
$ ./vendor/bin/drush en oe_showcase_page
```
## Usage
### Adding paragraphs to a page:

> To add paragraphs to a page, create a new page or edit an existing one and on the body field click on the dropdown arrow to reveal the enabed paragraphs types.

Reviewer: ```suggestion
To add paragraphs to a page, create a new page or edit an existing one, then on the body field click on the dropdown arrow to reveal the enabled paragraphs types.
```

> Select the desired pragraph ie Accordion (which contains several accordion items), fill out the required fields (Title and Body), to add a new Accordion item, click on [Add Accordion Item], to add other paragraphs types click on the dropdown and select the other component desired.

Reviewer: ```suggestion
Select the desired paragraph, e.g. Accordion (which contains several accordion items), fill out the required fields (Title and Body). To add a new Accordion item, click on [Add Accordion Item] and to add other paragraphs types click on the dropdown and select the desired component.
```

> You may also rearrange the order of each paragraph by dragging and dropping them using the left move cursor icon.

Reviewer: ```suggestion
The order of each paragraph can also be rearranged  by dragging and dropping them using the left move cursor icon.
```

### Configuration

> To add new paragraphs types to the body field other then the ones enabled by default, edit the body field by going on Administration > Structure > Content Types > OE Showcase Page > Manage Fields > Edit Body

Reviewer: ```suggestion
To add new paragraphs types to the body field (other then the ones enabled by default): Edit the body field by going to Administration > Structure > Content Types > OE Showcase Page > Manage Fields > Edit Body
```
